rest_framework_friendly_errors/mixins.py

```diff
@@ -203,6 +203,8 @@ def _run_validator(self, validator, field, message, parent=None):
             err_message = err.detail[0] \
                 if hasattr(err, 'detail') else err.message
             return err_message == message
+        except KeyError:
```

Comment:
Without a comment or reference, I'm not able to understand why this required (what's the problem it's solving?) and what are the side effects. Is there any consequence to this error failing silently?

Reply:
please refer : https://chat.awaaz.de/awaazde/pl/5o88izcqfin5bquauag8w79qrw

Reply:
I see KeyError needs to be caught, but my concern is you are failing silently. Isn't that a problem? Our thread doesn't explain fully to me whether what the right way to deal with this issue is. Maybe there's a deeper fix to prevent this key error in first place?

Reply:
@awaazde I m not sure, because there is no data for the key and thats why our code has raised error "Invalid advance option follow_up" from our validate method but this package is expecting key to be present since it is doing dat[key] while it is not present. Becuase of ProhibitNullCharactersValidator it is executing this code, which was intorduced now in django-2.0 also at the end our error messages will get raised properly.

Reply:
Why are you failing silently? Still not clear. Why not handle KeyError in same way other errors above are currently being handled?

Reply:
@awaazde As discussed returned False in case of KeyError